pyproject.toml

```diff
@@ -71,49 +71,161 @@ line-length = 79
 indent-width = 4
 
 target-version = "py38"
+exclude = [
+  "CSV",
+  "ref.*?py",
+]
 
 [tool.ruff.lint]
-fixable = ["ALL"]
-preview = true
+fixable = []
+# preview = true
+# logger-objects = []
+task-tags = ["TODO", "FIXME", "XXX"]
 select = [
+  "A", # flake8-builtins
+  "ARG", # flake8-unused-arguments
+  "COM", # flake8-commas
+  "D", # pydocstyle
   "E", # pycodestyle
+  "E701", # multiple-statements-on-one-line-colon
+  "E722", # bare-except
   "F", # Pyflakes
+  "F401", # unused-import
+  "G", # flake8-logging-format
+  "I", # isort
```

Comment:
isort just has two rules so we could just include I002 here and remove I001 from ignores.

Reply:
Yes we can. I suggest using this way. If any new rule is implemented in ruff's 'I' section, we get it; This reminds us that we ignore `I001` and might want to consider one day to conform with it.

Reply:
Yeah thats a good approach